android/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "ironoxide-android"
+version = "0.12.1"
```

Comment:
Not sure if we're starting this over at 0.1.0 or want it to be closer in line with the others?

Reply:
Done